src/App/Formatter/HXL/HXLAttributeCollection.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * Ushahidi API Formatter for HXL License
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Application
+ * @copyright  2014 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+namespace Ushahidi\App\Formatter\HXL;
+
+use Ushahidi\App\Formatter\API;
+use Ushahidi\App\Formatter\Collection;
+use Ushahidi\Core\Traits\FormatterAuthorizerMetadata;
+
+class HXLAttributeCollection extends Collection
```

Comment:
Do we actually need to extend the collection formatter?

Reply:
no, I thought I had deleted this. Sorry